oracle/datadog_checks/oracle/oracle.py

```diff
@@ -165,44 +149,40 @@ def check(self, _):
 
     @property
     def _connection(self):
```

Comment:
This is just a comment and is out of scope for this PR, but I think we should eventually abstract out the connection logic to another class, especially since the connection is conditional on either JDBC or oracle client.

Reply:
Yea, I think that was something I was planning on doing given more time but it definitely makes sense to do as a future PR.